usage/usage.py

```diff
@@ -143,16 +143,19 @@ def get_usage(data, res=100):
     end_times = [(rec['end'] - begin).total_seconds() for rec in data]
 
     node_counts = [int(rec['nnodes']) for rec in data]
+    job_type = [rec['jobname'] for rec in data]
 
     # Make a histogram representing number of used nodes in a given time
     # interval.
     usage = [0] * res
-    for begin, end, nodes in zip(start_times, end_times, node_counts):
+    names = [''] * res
```

Comment:
Since you have to split the strings you will create here later on in `usageplot.py` I think a list of lists would suit your needs better, i.e. replace current line with:
```python
names = [[] for _ in range(res)]
```